Failing with

```
java.lang.AssertionError: 
[Snapshot is not matching (use -Dsnap to update it automatically): CreateExtensionMojoIT/testCreateStandaloneExtension/my-org-my-own-ext_runtime_pom.xml] 
Path:
  target\test-classes\output\create-standalone-extension\my-org-my-own-ext\runtime\pom.xml
and path:
  src\test\resources\__snapshots__\CreateExtensionMojoIT\testCreateStandaloneExtension\my-org-my-own-ext_runtime_pom.xml
do not have same content:

Missing content at line 48:
  ["                                    <version>${quarkus.version}</version>"]

Extra content at line 51:
  ["                            <annotationProcessorPathsUseDepMgmt>true</annotationProcessorPathsUseDepMgmt>"]

	at io.quarkus.devtools.testing.SnapshotTesting.lambda$assertThatMatchSnapshot$0(SnapshotTesting.java:184)
	at io.quarkus.devtools.testing.SnapshotTesting.withSnapshotsDir(SnapshotTesting.java:98)
	at io.quarkus.devtools.testing.SnapshotTesting.assertThatMatchSnapshot(SnapshotTesting.java:161)
```


Will fix it asap

--- a/independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/extension-base/java/runtime/pom.tpl.qute.xml
+++ b/independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/extension-base/java/runtime/pom.tpl.qute.xml
@@ -59,13 +59,9 @@
                                 <path>
                                     <groupId>io.quarkus</groupId>
                                     <artifactId>quarkus-extension-processor</artifactId>
-                                    {#if quarkus.version}
-                                    <version>$\{quarkus.version}</version>
-                                    {#else}
-                                    <version>$\{project.version}</version>
-                                    {/if}
                                 </path>
                             </annotationProcessorPaths>
+                            <annotationProcessorPathsUseDepMgmt>true</annotationProcessorPathsUseDepMgmt>
                         </configuration>
                     </execution>
                 </executions>

